api-server/src/roster/urls.py

```diff
@@ -25,6 +25,13 @@
     path('aviators/fromuser/<int:user_id>/', AviatorFromUserListView.as_view()),
     path('prospective_aviators/detail/', ProspectiveAviatorDetailView.as_view()),
     path('stats/', StatsView.as_view()),
+    path('stats/flightlog/list/<aviator_pk>/', FlightLogListView.as_view()),
+    path('stats/flightlog/aggregate/<aviator_pk>/', FlightLogAggregateView.as_view()),
+    path('stats/flightlog/timeseries/<aviator_pk>/', FlightLogTimeSeriesView.as_view()),
+    path('stats/flightlog/timeseries/90days/<aviator_pk>/', FlightLogTimeSeries90DaysView.as_view()),
```

Comment:
@afuggetta the more i look at this i think i'd rather generalize both of the timeseries calls so that there is only one to maintain. I'll take for action.